packages/core/src/extensions/TableHandles/TableHandlesPlugin.ts

```diff
@@ -535,10 +546,16 @@ export class TableHandlesView<
     }
 
     // Hide handles if the table block has been removed.
-    this.state.block = this.editor.getBlock(this.state.block.id)!;
```

Comment:
TODO: In the old version we were potentially setting `this.state.block` to undefined. Now, it will keep the old (unavailable block). can this break anything?